site/reference/pkg/get/README.md

```diff
@@ -76,18 +74,24 @@ PKG_PATH:
 
 VERSION:
```

Comment:
Yeah, there are a few topics that are either concepts that doesn't seem to fit well within the reference docs, or that cuts across multiple commands (like the update strategies). I think for those we should rely on the book.

Reply:
Yeah, but I wanted to add those once we have the content ready. I want to have deep links to the actual relevant content instead of links to the top-level chapters.

Reply:
+1 on linking to book pages when ready.  I'd probably split the doc generation into several passes.  The cross linking can be left to the next pass.